pkg/cli/admin/upgrade/recommend/examples/4.12.16-longest-not-recommended.show-outdated-releases-output

```diff
@@ -0,0 +1,639 @@
+  
+  Pods scheduled to nodes running a RHCOS/kernel version with a known defect may fail to start with a CreateContainerError. https://issues.redhat.com/browse/COS-2437
+
+  Version: 4.13.0
+  Image: quay.io/openshift-release-dev/ocp-release@sha256:74b23ed4bbb593195a721373ed6693687a9b444c97065ce8ac653ba464375711
+  Reason: MultipleReasons
+  Message: Exposure to AcceleratedNetworkingRace is unknown due to an evaluation failure: client-side throttling: only 8m30.898043529s has elapsed since the last match call completed for this cluster condition backend; this cached cluster condition request has been queued for later execution
+  Adding a new worker node may fail for clusters running on Azure with Accelerated Networking. https://issues.redhat.com/browse/OPNET-479
+  
+  Exposure to MultiNetworkAttachmentsWhereaboutsVersion is unknown due to an evaluation failure: client-side throttling: only 8m30.898047129s has elapsed since the last match call completed for this cluster condition backend; this cached cluster condition request has been queued for later execution
+  Upgrade can get stuck on clusters that have multiple network attachments. https://access.redhat.com/solutions/7024726
+  
+  Exposure to PerformanceProfilesCPUQuota is unknown due to an evaluation failure: client-side throttling: only 8m30.898050029s has elapsed since the last match call completed for this cluster condition backend; this cached cluster condition request has been queued for later execution
+  Pods may not execute properly because of CPU quota issues on clusters using PerformanceProfiles. https://issues.redhat.com/browse/OCPNODE-1705
+  
+  New PersistentVolumes are created bound to an invalid symlink instead of the correct one and thus fail once updated to 4.13.4 or later as the invalid symlink dispears. https://issues.redhat.com/browse/COS-2349
+  
+  Pods scheduled to nodes running a RHCOS/kernel version with a known defect may fail to start with a CreateContainerError. https://issues.redhat.com/browse/COS-2437
+
+  VERSION     IMAGE
+  4.12.64     quay.io/openshift-release-dev/ocp-release@sha256:669170342e3ae3456b2eb00dd0c45cd817d62af310e0aa2bf214f1da65fae9d0
+  4.12.63     quay.io/openshift-release-dev/ocp-release@sha256:5db6f8dd1db6b9d07dacfa74574a38a6e518145a3c0ab5d895e9c89e029a39e4
+  4.12.61     quay.io/openshift-release-dev/ocp-release@sha256:faa201ea7790bb101de7b9b6f01543136e8c589463b8437a10678da0cdd0197a
+  4.12.60     quay.io/openshift-release-dev/ocp-release@sha256:22174072f8aaef46f80bded25d5a2bdfa6fd8bb01fc5242a07e550f132cafc5a
+  4.12.59     quay.io/openshift-release-dev/ocp-release@sha256:8908e5a46f106e7520faa5f5ee679033c4c2f0026f1b8a339e6b3ea4becdd969
+  4.12.58     quay.io/openshift-release-dev/ocp-release@sha256:b622d947ade4a13754e9d9f6803107f1b321eecd925c897f3c12de1aafccbdc5
+  4.12.57     quay.io/openshift-release-dev/ocp-release@sha256:ebd20cd66e0bbb5bcd7d16559ff4856918b7172e29d6a7bd34d8cc49a556b8f7
```

Comment:
I agree with your volume concerns ([also these][1]).  Follow-up work planned in [OTA-1273](https://issues.redhat.com/browse/OTA-1273) will add `--to` and give us the option to avoid rendering messages for some releases.  For example, we could go down to one-line-per-target tables, and have a new column for matching risk slugs.  But for now, without `--to`, I think the volume is acceptable while we step towards a more complete solution.  Because part of the push here is to put at least the longest-hop risk `message` in front of the admin, without them having to opt-in to see that text.

[1]: https://github.com/openshift/oc/pull/1889#pullrequestreview-2347863330